massa-versioning/src/grpc_mapping.rs

```diff
@@ -15,13 +15,23 @@ impl From<&ComponentStateTypeId> for grpc::ComponentStateId {
     }
 }
 
+impl From<&MipComponent> for grpc::MipComponent {
+    fn from(value: &MipComponent) -> Self {
+        match value {
+            MipComponent::KeyPair => grpc::MipComponent::Keypair,
+            MipComponent::Address => grpc::MipComponent::Address,
+            _ => grpc::MipComponent::Unspecified,
+        }
+    }
+}
+
```

Comment:
IMHO, if you are mirroring your Rust struct and  proto ones, just use the identifier without explicite mapping. 

The coherence is assured by the fact that we always must add new enums at the end of the enum. There is a CI in proto files to detect breaking changes

Reply:
My bad, It's a new case and It's totally different than existing mappings. So, this the way. I have spoken